### DISCUSSION TITLE
Placement of quick-start.md

### DISCUSSION THREAD

**Discussion by benwandrew** [AutoResearch/autora#389] [Core Contributions]
If we have more explanatory markdown files than just `index.md` (eg, meta parameters, search space, etc), does it make most sense to have all of those sequenced first in the `nav` in `mkdocs.yml`, and then `quick-start.md` immediately preceding the `examples.ipynb`? that's how i read the template, and what i what do intuitively, but just wanted to confirm.

**Comment by younesStrittmatter** [comment]
I thought otherwise:
- index.md
- quick-start.md
- ...
- ...
- Examples:
   - basic-usage.ipynb

But maybe it makes more sense, to do it like you said? What do the others think?

**Comment by benwandrew** [comment]
i was just thinking of having all the conceptual bits together, and then all the practical ones. as in, here is what this contribution is/does, and now here is how to get hands-on and use it (starting with quickstart installation and proceeding to examples). but i'm open as well

**Reply by musslick** [reply]
Hmm... I do think it would be nice to have the quickstart guide early on. This is because I don't believe people will be reading through all the "content" materials before going to the quickstart. It will likely just be "index.md" (getting a sense of what this thing is) -> "quickstart.md".

**Reply by benwandrew** [reply]
sounds good, i think we’ve settled on keeping it immediately after `index.md`
